consensus/manager_test.go

```diff
@@ -42,16 +68,16 @@ func TestManager(t *testing.T) {
 	genDoc := genesis.MakeGenesis(getTime, accs, vals, params)
 
 	rewardAddrs := []crypto.Address{
-		ts.RandomAddress(), ts.RandomAddress(),
-		ts.RandomAddress(), ts.RandomAddress(),
-		ts.RandomAddress(),
+		ts.RandAddress(), ts.RandAddress(),
+		ts.RandAddress(), ts.RandAddress(),
+		ts.RandAddress(),
 	}
 	signers := make([]crypto.Signer, 5)
 	signers[0] = committeeSigners[0]
```

Comment:
do we sure that `committeeSigners` has any data? if not we should check the length of that otherwise the program will panic